L1Trigger/TrackerDTC/interface/LayerEncoding.h

```diff
@@ -23,10 +23,13 @@ namespace trackerDTC {
     ~LayerEncoding() {}
     // decode layer id for given sensor module
     int decode(tt::SensorModule* sm) const;
+    // get encoded layers read by given DTC
+    const std::vector<int>& layers(int dtcId) const {return encodingsLayerId_.at(dtcId%numDTCsPerRegion_);} 
 
   private:
     // helper class to store configurations
     const tt::Setup* setup_;
+    int numDTCsPerRegion_;
```

Comment:
Please add comment to new data member.

Reply:
done

Reply:
A detector phi nonant = 24. Now clarified comment.
It corresponds to Setup::numDTCsPerRegion()